desktop-app/app/components/ErrorBoundary/index.js

```diff
@@ -1,4 +1,16 @@
 import React, {Fragment} from 'react';
+import TextBoxWithCopyButton from '../../utils/TextBoxWithCopyButton';
+import Button from '@material-ui/core/Button';
+
+const {openNewGitHubIssue} = require('electron-util');
+
+const createIssue = () => {
+  openNewGitHubIssue({
+    user: 'responsively-org',
+    repo: 'responsively-app',
+    body: '',
```

Comment:
should we fill the body with some info?
- System environment, app version, etc (what is displayed in About dialog)
- The actual error and stack trace?

It could be something like this:

>Hi I'm reporting an app crash:
>>Version: 0.11.0
>>Electron: 9.1.1
>>Chrome: 83.0.4103.122
>>Node.js: 12.14.1
>>V8: 8.3.110.13-electron.0
>>OS: Windows_NT x64 10.0.18362
>
>**Error Info:** `{....}`
>**Stack Trace:** `{....}`

Maybe we should think about download that info (Error Info and Stack Trace) into separate files to attach them into the github issue because that could be a big message to send right? @manojVivek

Reply:
Yes, I agree we should populate the body with app `About` details, actual error and stack trace. 

But we can skip the download option and just add a copy button, as I don't think the stack trace and error info is going to be so huge. Maybe some 10 lines each, so we can skip the download option.